core/src/main/scripts/importer/validateData.py

```diff
@@ -3025,6 +3454,15 @@ def process_metadata_files(directory, portal_instance, logger, relaxed_mode):
                            'cause': stable_id})
             else:
                 stable_ids.append(stable_id)
+        # Validate datatype, for segmentation file only one can be loaded
+        if 'datatype' in meta_dictionary:
+            if meta_dictionary['datatype'].lower() == "seg":
+                if study_data_type is not None:
+                    logger.error(
+                        'datatype SEG is repeated. Only one segmentation file can be loaded.')
+                else:
+                    study_data_type = meta_dictionary['datatype']
```

Comment:
`study_data_type` is not a proper name to check "presence of segment datafile". 

What about making a list, like the stable_ids list, such as study_data_types.
```
study_data_types = []
if 'datatype' in meta_dictionary:
    if meta_dictionary['datatype'].lower() in study_data_types:
        if meta_dictionary['datatype'].lower() == "seg":
            error......
    else:
         study_data_types.append(meta_dictionary['datatype'].lower())